docs/mp/integrations/wls.adoc

```diff
@@ -0,0 +1,472 @@
+
+mp:
+  messaging:
+    connector:
+      helidon-jms:
+        jndi:
+           jms-factory: qcf #<3>
+           env-properties: #<4>
+              java.naming.factory.initial: weblogic.jndi.WLInitialContextFactory
+              java.naming.provider.url: ${wls-admin-url}
+              java.naming.security.principal: ${wls-username}
+              java.naming.security.credentials: ${wls-password}
+
+    incoming:
+      from-wls-q:
+        connector: helidon-jms
+        jndi.destination: queuejndi #<5>
+        type: queue
+
+    outgoing:
+      to-wls-q:
+        connector: helidon-jms
+        jndi.destination: queuejndi #<5>
+        type: queue
+----
+<1> Admin server t3 connection URL
+<2> Example of WLS Cluster t3 connection URL
+<3> Connection factory name
+<4> JMS environment properties to lookup resources
+<5> Destination with `jndi.` prefix is evaluated as JNDI name, simple `destination` is evaluated as a CDI syntax
+
+For more detailed reading about Reactive Messaging configuration see xref:../../mp/reactivemessaging/introduction.adoc#_configuration[Reactive Messaging Configuration].
+
+After you have configured the Helidon JMS connector,
+you can use xref:../../mp/reactivemessaging/introduction.adoc[Helidon Reactive Messaging] for consuming and sending messages.
+
+[source, java]
+.Consuming messages from Oracle Weblogic JMS Server
+----
+@Incoming("from-wls-q")
+public void receive(String msg) {
+    System.out.println("Process JMS message as per business logic" + msg);
+}
+----
+
+[source, java]
+.Producing messages to Oracle Weblogic JMS Server
+----
+@Outgoing("to-wls-q")
+public PublisherBuilder<String> produceToJms() {
+    return ReactiveStreams.of("test1", "test2");
+}
+----
+
+For more information about setting up JMS integration between Helidon and Oracle Weblogic Server, see {wls-jms-link}.
+
+=== References [[JMS-Integration-References]]
+
+* xref:../../mp/reactivemessaging/introduction.adoc[Reactive Messaging]
+* xref:../../mp/reactivemessaging/jms.adoc[JMS Connector]
+* xref:../../mp/reactivemessaging/weblogic.adoc[WebLogic JMS Connector for legacy thin T3 clients]
+* {osdc-link}
+* {wls-jms-link}
+
+== Web Services Integration [[Web-Services-Integration]]
+
+- <<Web-Services-Integration-Overview, Overview>>
+- <<Web-Services-Integration-Maven-Coordinates, Maven Coordinates>>
+- <<Web-Services-Integration-Usage, Usage>>
+- <<Web-Services-Integration-Example, Example>>
+- <<Web-Services-Integration-References, References>>
+
+=== Overview [[Web-Services-Integration-Overview]]
+
+Helion MP and WebLogic Server Web Services integration enables the Helidon microservice application to communicate with the WebLogic Web Service deployed in Oracle WebLogic Server.
+
+=== Maven Coordinates [[Web-Services-Integration-Maven-Coordinates]]
+
+Add the `com.oracle.webservices.wls.jaxws-wlswss-client.jar` client file downloaded from {osdc-link}, as part of the Maven dependencies:
```

Comment:
I recommend adding an internal link back to the maven install example, provided in the JMS section previously (in case a reader just starts at this section b/c they are not interested in JMS integration)